backend/services/feed.ts

```diff
@@ -38,45 +55,44 @@ async function getPostsForBlockRange({ from, to }: BlockRange): Promise<[number,
   return posts;
 }
```

Comment:
Suggestion: reverse ordering the feed will be better for demos, also more standard
In testing, this works, but I don't know if there are any edge cases where it might break.
```suggestion
  return posts.reverse();
```